### ISSUE TITLE
Freezes by using force feedback - Kernel bug?

### ISSUE THREAD

**Issue by fizzgigfuzzlehead** [ValveSoftware/steamos_kernel#27]
Hey guys,

I got strange freezes while playing DiRT: Showdown with my wireless XBox 360 controller.

I posted this first here at Virtual Programming:
https://github.com/virtual-programming/dirtshowdown-linux/issues/5#issuecomment-139026856

Is it true that this could be a Kernel bug? Who would be able to fix this?

**Comment by ghost** [comment]
Hi
The relevant part was this appearing in the kernel log:

```
Sep 6 12:55:31 steamos kernel: [ 287.145066] WARNING: at /usr/src/packages/BUILD/linux-3.10.60/drivers/usb/core/urb.c:327 usb_submit_urb+0x4e/0x2ba usbcore
Sep 6 12:55:31 steamos kernel: [ 287.145067] URB ffff880234edfb40 submitted while active
Sep 6 12:55:31 steamos kernel: [ 287.145068] Modules linked in: bnep rfcomm nls_utf8 nls_cp437 vfat fat snd_hda_codec_hdmi arc4 acpi_cpufreq mperf kvm_amd kvm crc32c$
Sep 6 12:55:31 steamos kernel: [ 287.145095] CPU: 2 PID: 4477 Comm: dirt.i386 Tainted: P O 3.10-5-amd64 #1 SteamOS 3.10.60-1+steamos1
Sep 6 12:55:31 steamos kernel: [ 287.145097] Hardware name: Gigabyte Technology Co., Ltd. To be filled by O.E.M./F2A88XN-WIFI, BIOS F4 05/28/2014
Sep 6 12:55:31 steamos kernel: [ 287.145098] ffffffff8138fb4c 0000000000000000 ffffffff8103cf89 ffff8800be45e360
Sep 6 12:55:31 steamos kernel: [ 287.145101] ffff880234443cc8 ffffffff8104908d 00000000fffffff0 0000000000000010
Sep 6 12:55:31 steamos kernel: [ 287.145103] ffff880231b8e400 00000000ffffff80 ffffffff8103d039 ffffffffa001df7a
Sep 6 12:55:31 steamos kernel: [ 287.145105] Call Trace:
Sep 6 12:55:31 steamos kernel: [ 287.145110] [] ? dump_stack+0xd/0x17
Sep 6 12:55:31 steamos kernel: [ 287.145114] [] ? warn_slowpath_common+0x5f/0x77
Sep 6 12:55:31 steamos kernel: [ 287.145117] [] ? internal_add_timer+0xd/0x28
Sep 6 12:55:31 steamos kernel: [ 287.145119] [] ? warn_slowpath_fmt+0x45/0x4a
Sep 6 12:55:31 steamos kernel: [ 287.145127] [] ? usb_submit_urb+0x4e/0x2ba [usbcore]
Sep 6 12:55:31 steamos kernel: [ 287.145130] [] ? ml_play_effects+0x3b/0x383 [ff_memless]
Sep 6 12:55:31 steamos kernel: [ 287.145132] [] ? ml_ff_playback+0x82/0x8c [ff_memless]
Sep 6 12:55:31 steamos kernel: [ 287.145135] [] ? input_ff_event+0x7c/0x80
Sep 6 12:55:31 steamos kernel: [ 287.145137] [] ? input_handle_event+0x367/0x44b
Sep 6 12:55:31 steamos kernel: [ 287.145139] [] ? input_inject_event+0x69/0x91
Sep 6 12:55:31 steamos kernel: [ 287.145142] [] ? should_resched+0x5/0x23
Sep 6 12:55:31 steamos kernel: [ 287.145145] [] ? evdev_write+0x89/0xc3 [evdev]
Sep 6 12:55:31 steamos kernel: [ 287.145148] [] ? spin_unlock+0x5/0x6
Sep 6 12:55:31 steamos kernel: [ 287.145151] [] ? vfs_write+0x9d/0x103
Sep 6 12:55:31 steamos kernel: [ 287.145154] [] ? SyS_write+0x51/0x80
Sep 6 12:55:31 steamos kernel: [ 287.145156] [] ? ia32_do_call+0x13/0x13
Sep 6 12:55:31 steamos kernel: [ 287.145157] ---[ end trace c7faa89af51267c4 ]---
```

when the user disabled force feedback in game, the problem went away. We cannot replicate this ourselves on Ubuntu 15.04, and we've had no other reports of it so far.

**Comment by Plagman** [comment]
This is fixed in SteamOS 2.0, I recommend all users update to brewmaster ASAP.

**Comment by ghost** [comment]
OK, cool.. we will advise that :)
